Dockerfile

```diff
@@ -1,74 +1,27 @@
-FROM  node:14.15-alpine AS assets-builder
-
-RUN apk --no-cache add ca-certificates make
-
-WORKDIR /app
-
-COPY package.json yarn.lock ./
-COPY assets/. ./assets/
-COPY tailwind.config.js ./
-
-ADD .env.example ./.env
-COPY Makefile ./Makefile
-
-RUN yarn
-
-# Prepare all assets
-RUN make assets
-
-
-FROM golang:1.15-alpine AS migration
+FROM golang:1.18-alpine
 
 ARG DATABASE_URL
 
-# Move to working directory /migration
-WORKDIR /migration
-
-# Set necessary environment variables needed for our image
-ENV GO111MODULE=on \
-    CGO_ENABLE=0 \
-    GOOS=linux \
-    GOARCH=amd64
-
-# Copy the code into the container
-COPY . .
-
-# Install command-line tool
-RUN go get github.com/beego/bee/v2
-
-# Migrate database
-RUN bee migrate -driver=postgres -conn=$DATABASE_URL
-
-
-FROM golang:1.15-alpine
-
-# Set necessary environment variables needed for our image
-ENV GO111MODULE=on \
-    CGO_ENABLE=0 \
-    GOOS=linux \
-    GOARCH=amd64
+ENV GIN_MODE=release
 
 WORKDIR /app
 
-# Copy go mod and sum files
+# Install Go dependencies
 COPY go.mod go.sum ./
-
-# Download all dependancies
 RUN go mod download
 
-# Copy ENV file
-ADD .env.example ./.env
+# Install goose for migration
+RUN go install github.com/pressly/goose/v3/cmd/goose@latest
 
-# Copy the code into the container
+# Copy codebase
 COPY . .
 
-# Copy assets from assets builder
-COPY --from=assets-builder /app/static/. ./static/
+# Run the migration
+RUN goose -dir database/migrations -table "migration_versions" postgres "$DATABASE_URL" up
```

Comment:
I know this came from gin-templates, but I think the migration should not be run when building the docker container. It should be run before starting the application instead.
How about creating a `./bin/start.sh` file, then moving this migration command and `./main` into it? 💭

Reply:
moved in [0d9f041](https://github.com/carryall/go-google-scraper-challenge/pull/80/commits/0d9f041b6b936a1f5eae15b1a52fa07fe7c23f4c)